open_city_profile/tests/conftest.py

```diff
@@ -102,3 +104,13 @@ def gql_schema(rf, anon_user_gql_client):
 def mock_responses():
     with responses.RequestsMock(assert_all_requests_are_fired=False) as mock_resps:
         yield mock_resps
+
+
+def unix_timestamp_now():
+    epoch = datetime(1970, 1, 1, tzinfo=timezone.utc)
+    return int((datetime.now(tz=timezone.utc) - epoch).total_seconds())
+
+
+@pytest.fixture(name="unix_timestamp_now")
+def unix_timestamp_now_fixture():
```

Comment:
After not needing unix_timestamp_now function separately I would rename this. For not needing to set the name manually.